bundle/config/resources_test.go

```diff
@@ -200,6 +211,9 @@ func TestResourcesBindSupport(t *testing.T) {
 
 			// bind operation requires Exists to return true
 			exists, err := r.Exists(ctx, m.WorkspaceClient, "0")
+			if err != nil {
+				panic(err)
+			}
```

Comment:
Why?

Reply:
a leftover from debugging - remove this in a follow up commit